centreon/www/class/centreonConfigCentreonBroker.php

```diff
@@ -745,6 +746,11 @@ public function insertConfig(array $values): bool
                 (int) $this->checkEventMaxQueueSizeValue($values['event_queue_max_size']),
                 \PDO::PARAM_INT
             );
+            $stmt->bindValue(
+                ':event_queues_total_size',
+                ! empty($values['event_queues_total_size']) ? (int) $values['event_queues_total_size'] : null,
```

Comment:
Suggestion of "light syntax", but a bit tricky for those who does not understand well the php syntax avec operator precedence.
```suggestion
                (int)($values['event_queues_total_size'] ?? 0) ?: null,
```

Reply:
I take note of this but this PR is a backport, I will keep code unchanged